testsuite/MDAnalysisTests/test_atomselections.py

```diff
@@ -464,6 +464,13 @@ def test_nucleic(self):
         assert_equal(rna.n_atoms, 739)
         assert_equal(rna.n_residues, 23)
 
+    def test_nucleic_all(self):
+        u = mda.Universe(NUCLsel)
+
+        sel = u.select_atoms('nucleic')
+
+        assert_(len(sel) == len(u.atoms))
```

Comment:
It could be worth adding one atom that should not be selected to be sure the `nucleic` token is not just selecting everything regardless of the resname.